### PR TITLE
Improve local testing for all RDBMS

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+test-all: vendor/bin/phpunit --testsuite Sqlite,Mysql,Pgsql,Mssql,Oracle
+test-sqlite: vendor/bin/phpunit --testsuite Sqlite
+test-mysql: vendor/bin/phpunit --testsuite Mysql
+test-pgsql: vendor/bin/phpunit --testsuite Pgsql
+test-mssql: vendor/bin/phpunit --testsuite Mssql
+test-oracle: vendor/bin/phpunit --testsuite Oracle
```
Comment (samdark) [inline Makefile:6]: I'd ensure that database container is up here.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+RUN pecl install sqlsrv
+RUN printf "; priority=20\nextension=sqlsrv.so\n" > /usr/local/etc/php/conf.d/php-sqlsrv.ini
+
+RUN pecl install pdo_sqlsrv
+RUN printf "; priority=30\nextension=pdo_sqlsrv.so\n" > /usr/local/etc/php/conf.d/php-pdo-sqlsrv.ini
+
+# Composer
+
+COPY --from=composer:2.8.1 /usr/bin/composer /usr/local/bin/composer
```
Comment (samdark) [inline Dockerfile:71]:
```suggestion
COPY --from=composer/composer:latest-bin:2.8.1 /composer /usr/bin/composer
```

Could reduce copying a bit but it's OK as is.

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  mysql:
+    image: mysql:9
+    ports:
+      - "3306:3306"
+    volumes:
+      - type: tmpfs
+        target: /var/lib/mysql
+    environment:
+      MYSQL_DATABASE: yii
+      MYSQL_ROOT_PASSWORD: root
```
Comment (samdark) [inline docker-compose.yml:23]: How about unifying username/password for all databases?

**Review by samdark — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (arogachev) [reply Dockerfile]:
```
failed to solve: failed to parse stage name "composer/composer:latest-bin:2.8.1": invalid reference format
```

Comment (samdark) [reply Dockerfile:71]: Hmm... seems you need a separate FROM as in here: https://github.com/yiisoft/app/blob/master/docker/Dockerfile but it's OK as is.

File: docker-compose.yml (hunk 3 above)
Comment (arogachev) [reply docker-compose.yml:23]: #76

**Comment by samdark** [comment]
It's good at this stage, I think. Let's merge it.

**Pull request merged**